stuckTransfers.js

```diff
@@ -1,107 +1,108 @@
-const web3Foreign = new Web3(foreignProvider);
+const { FOREIGN_RPC_URL, FOREIGN_BRIDGE_ADDRESS, ERC20_ADDRESS } = process.env
+const FOREIGN_DEPLOYMENT_BLOCK = Number(process.env.FOREIGN_DEPLOYMENT_BLOCK) || 0
 
-const ABI_TransferWithoutData = [
+const foreignProvider = new Web3.providers.HttpProvider(FOREIGN_RPC_URL)
+const web3Foreign = new Web3(foreignProvider)
+
+const ABITransferWithoutData = [
   {
-    "anonymous": false,
-    "inputs": [
+    anonymous: false,
+    inputs: [
       {
-        "indexed": true,
-        "name": "from",
-        "type": "address"
+        indexed: true,
+        name: 'from',
+        type: 'address'
       },
       {
-        "indexed": true,
-        "name": "to",
-        "type": "address"
+        indexed: true,
+        name: 'to',
+        type: 'address'
       },
       {
-        "indexed": false,
-        "name": "value",
-        "type": "uint256"
+        indexed: false,
+        name: 'value',
+        type: 'uint256'
       }
     ],
-    "name": "Transfer",
-    "type": "event"
+    name: 'Transfer',
+    type: 'event'
   }
 ]
 
-const ABI_withData = [{
-  "anonymous": false,
-  "inputs": [
-    {
-      "indexed": true,
-      "name": "from",
-      "type": "address"
-    },
-    {
-      "indexed": true,
-      "name": "to",
-      "type": "address"
-    },
-    {
-      "indexed": false,
-      "name": "value",
-      "type": "uint256"
-    },
-    {
-      "indexed": false,
-      "name": "data",
-      "type": "bytes"
-    }
-  ],
-  "name": "Transfer",
-  "type": "event"
-}]
+const ABIWithData = [
+  {
+    anonymous: false,
+    inputs: [
+      {
+        indexed: true,
+        name: 'from',
+        type: 'address'
+      },
+      {
+        indexed: true,
+        name: 'to',
+        type: 'address'
+      },
+      {
+        indexed: false,
+        name: 'value',
+        type: 'uint256'
+      },
+      {
+        indexed: false,
+        name: 'data',
+        type: 'bytes'
+      }
+    ],
+    name: 'Transfer',
+    type: 'event'
+  }
+]
 
-function compareTransfers(transfersNormal){
-  return function(withData){
-    return transfersNormal.filter(function(normal){
-      return normal.transactionHash === withData.transactionHash
-    }).length == 0;
+function compareTransfers(transfersNormal) {
+  return withData => {
+    return (
+      transfersNormal.filter(normal => {
+        return normal.transactionHash === withData.transactionHash
+      }).length === 0
+    )
   }
 }
 
-async function main(){
+async function main() {
   try {
-    const tokenContract = new web3Foreign.eth.Contract(ABI_TransferWithoutData, POA20_ADDRESS);
-    const tokenContractWithData = new web3Foreign.eth.Contract(ABI_withData, POA20_ADDRESS);
-    logger.debug('calling tokenContract.getPastEvents Transfer');
-    let transfersNormal = await tokenContract.getPastEvents('Transfer', {
+    const tokenContract = new web3Foreign.eth.Contract(ABITransferWithoutData, ERC20_ADDRESS)
```

Comment:
Since this module is not used so far please add a comment that the address needs to be extracted from the bridge contract.